user_docs/en/changes.t2t

```diff
@@ -7,12 +7,12 @@ What's New in NVDA
 This release includes optional experimental support for UIA in Excel and Chromium browsers.
 There are fixes for several languages, and for accessing links in Braille.
 There are updates to Unicode CLDR and mathematical symbols, eSpeak-NG and LibLouis.
-
 Many bug fixes and improvements, including in Office, Visual Studio and several languages.
-Note: this is an Add-on API compatibility breaking release.
-Add-ons will need to be re-tested and have their manifest updated.
 This release also drops support for Adobe Flash.
 
+Note: this is an Add-on API compatibility breaking release.
+Add-ons will need to be re-tested and authors will need to provide a new version certifying its compatibility with this release.
+

```

Comment:
```suggestion

Note:
 - This release breaks compatibility with existing add-ons.
 - This release also drops support for Adobe Flash.

```

Reply:
Thanks @feerrenrut - I'm in favour of that wording.

Reply:
Thanks @feerrenrut, this wording looks good.